packages/material-ui/src/BottomNavigationAction/BottomNavigationAction.js

```diff
@@ -56,57 +55,56 @@ export const styles = theme => ({
   },
 });
 
-class BottomNavigationAction extends React.Component {
-  handleChange = event => {
-    const { onChange, value, onClick } = this.props;
+const BottomNavigationAction = React.forwardRef(function BottomNavigationAction(props, ref) {
+  const {
+    classes,
+    className: classNameProp,
+    icon,
+    label,
+    onChange,
+    onClick,
+    selected,
+    showLabel: showLabelProp,
```

Comment:
```suggestion
    showLabel,
```

Reply:
I see zero gain at renaming the property.

Reply:
Good catch. I don't know why that was done, but 'blame' says it was you that did it :-)